cogs/tags/tags.py

```diff
@@ -183,7 +195,7 @@ async def syncAllowedRoles(self):
             guild = discord.utils.get(self.bot.guilds, id=guildId)
             async with self.configV3.guild(guild).tiers() as tiers:
                 self.allowed_roles[guildId] = set(tiers.keys())
-        # TODO debug logging print(self.allowed_roles.keys())
+        self.logger.debug("Roles allowed to create commands: %s", self.allowed_roles.keys())
```

Comment:
Nit, this might look a bit ugly since it will probably have 
`self.allowed_roles([item 1, item 2, item3])`
as output

Maybe convert everything to a string and use 
`" ,".join(keyList)`

Reply:
Good point, fixed.